bricksrc/definitions.csv

```diff
@@ -989,26 +954,7 @@ https://brickschema.org/schema/Brick#Supply_Air_Temperature_Setpoint,Temperature
 https://brickschema.org/schema/Brick#Supply_Air_Temperature_Setpoint_Limit,A parameter limiting a Supply_Air_Temperature_Setpoint,
 https://brickschema.org/schema/Brick#Supply_Air_Temperature_Step_Parameter,,
 https://brickschema.org/schema/Brick#Supply_Air_Velocity_Pressure_Sensor,,
-https://brickschema.org/schema/Brick#Supply_Chilled_Water,,
-https://brickschema.org/schema/Brick#Supply_Chilled_Water_Temperature_Setpoint,Temperature setpoint for supply chilled water,
-https://brickschema.org/schema/Brick#Supply_Condenser_Water,"In a condenser water loop, this is water being brought to the condenser side of a heat-rejection device (e.g. chiller). It is the 'cold' side.",
-https://brickschema.org/schema/Brick#Supply_Condenser_Water_Flow_Sensor,Measures the flow of the supply condenser water,
-https://brickschema.org/schema/Brick#Supply_Condenser_Water_Temperature_Sensor,Measures the temperature of the supply condenser water,
-https://brickschema.org/schema/Brick#Supply_Condenser_Water_Temperature_Setpoint,The temperature setpoint for the supply condenser water,
 https://brickschema.org/schema/Brick#Supply_Fan,Fan moving supply air -- air that is supplied from the HVAC system into the building,
-https://brickschema.org/schema/Brick#Supply_Hot_Water,,
-https://brickschema.org/schema/Brick#Supply_Hot_Water_Temperature_Setpoint,Temperature setpoint for supply hot water,
-https://brickschema.org/schema/Brick#Supply_Water,,
-https://brickschema.org/schema/Brick#Supply_Water_Differential_Pressure_Deadband_Setpoint,Sets the size of a deadband of differential pressure of supply water,
-https://brickschema.org/schema/Brick#Supply_Water_Differential_Pressure_Integral_Time_Parameter,,
-https://brickschema.org/schema/Brick#Supply_Water_Differential_Pressure_Proportional_Band_Parameter,,
-https://brickschema.org/schema/Brick#Supply_Water_Flow_Sensor,Measures the rate of flow of hot supply water,
-https://brickschema.org/schema/Brick#Supply_Water_Flow_Setpoint,Sets the flow rate of hot supply water,
-https://brickschema.org/schema/Brick#Supply_Water_Temperature_Alarm,An alarm that indicates the off-normal conditions associated with temperature of the supply water.,
-https://brickschema.org/schema/Brick#Supply_Water_Temperature_Deadband_Setpoint,Sets the size of a deadband of temperature of supply water,
-https://brickschema.org/schema/Brick#Supply_Water_Temperature_Integral_Time_Parameter,,
-https://brickschema.org/schema/Brick#Supply_Water_Temperature_Proportional_Band_Parameter,,

```

Comment:
Where did you add the definition for the new classes?

Reply:
forgot to commit, whoops!